app/Shell/Docker.php

```diff
@@ -139,6 +139,13 @@ public function isDockerServiceRunning(): bool
 
     public function stopDockerService(): void
     {
-        $this->shell->execQuietly("test -z $(docker ps -q 2>/dev/null) && osascript -e 'quit app \"Docker\"'");
+        if (PHP_OS_FAMILY === 'Windows') {
+            $this->shell->execQuietly("wsl -t docker-desktop");
```

Comment:
I just tested this command out on WSL and it failed with this message:
![image](https://user-images.githubusercontent.com/2329654/95300155-a59fce00-0833-11eb-8d65-bb4579173b64.png)

Am I doing something wrong or does the `wsl` command need to be installed?

Reply:
Hi @josecanhelp,

**wsl** is a windows command that should be executed from within the windows shell. It will not be available from within the Linux environment.

Reply:
@josecanhelp A pre-requisite requirement to use takeout tool in Windows environment is to have WSL2 enabled/installed. Please click [here](https://github.com/tighten/takeout#requirements) to check the requirements section of the project.

Reply:
@josecanhelp I just wanted to cross check if your Windows environment has the wsl2 setup, as it is one of the [requirements](https://github.com/tighten/takeout#requirements) of takeout application.
Also i tested the stop docker service commands involving WSL(as given in the PR) on Windows powershell and command line (as @jonsugar pointed out). Please do let me know if I am proceeding in the wrong direction to solve this issue.